json_schema/project.schema.json

```diff
@@ -0,0 +1,310 @@
+            "additionalProperties": false,
+            "required": [
+              "team",
+              "id",
+              "name",
+              "members"
+            ],
+            "properties": {
+              "team": {
+                "const": true,
+                "description": "Indicates whether or not this is a Team. Must be 'true' for Teams."
+              },
+              "id": {
+                "type": "string",
+                "description": "A unique identifier of this Team."
+              },
+              "name": {
+                "type": "string",
+                "description": "A human-readable name of this Team."
+              },
+              "uri": {
+                "type": "string",
+                "description": "A URI users can visit to view more about this Team.",
+                "format": "uri"
+              },
+              "members": {
+                "type": "array",
+                "description": "A list of authors that are part of this t Team.",
+                "items": {
+                  "$ref": "#/definitions/author"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "tags": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "description": "A tag that can be used to categorize this Project.",
+        "examples": [
+          "mod",
+          "modpack",
+          "resourcepack",
+          "datapack",
+          "world",
+          "framework"
+        ]
+      }
+    },
+    "license": {
+      "type": "object",
+      "description": "License Information for this Project.",
+      "additionalProperties": false,
+      "required": [
+        "id",
+        "name",
+        "uri"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "An identifier for this license. Includes all SPDX identifiers, as well as ARR, MMPL, Generic-Open, and Generic-Proprietary."
+        },
+        "name": {
+          "type": "string",
+          "description": "A human-readable name of this license."
+        },
+        "uri": {
+          "type": "string",
+          "description": "A URI where the full text of this license can be viewed.",
+          "format": "uri"
+        },
+        "modpackUsage": {
+          "type": "string",
+          "description": "Information on whether this Project can be used in modpacks.",
+          "enum": [
+            "alwaysAllow",
+            "requiresPermission",
+            "neverAllow"
+          ]
+        }
+      }
+    },
+    "links": {
+      "type": "array",
+      "description": "A list of links relating to this Project.",
+      "items": {
+        "type": "object",
+        "description": "A Link Object related to the Project.",
+        "required": [
+          "rel",
+          "name",
+          "uri"
+        ],
+        "properties": {
+          "rel": {
+            "oneOf": [
```

Comment:
I think it might be a really rare case that someone wants to link to the same page with different relation names. Maybe this could be simplified by only allowing a string.

Reply:
I think there are enough circumstances where it could be an array. In the example, the values `x-discord` and `community` are used. For a non-standard value (like `x-discord`) it's helpful to pair it with a standard value (like `community`).

Reply:
That sounds reasonable.